test/foundry/mocks/policy/MockRoyaltyPolicyLAP.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+
+import { IRoyaltyPolicyLAP } from "../../../../contracts/interfaces/modules/royalty/policies/IRoyaltyPolicyLAP.sol";
+
+contract MockRoyaltyPolicyLAP is IRoyaltyPolicyLAP {
```

Comment:
Do we still need to create this new mock?

Reply:
See [this thread](https://github.com/storyprotocol/protocol-core-v1/pull/36#discussion_r1554431695) of resolved comment above

Notably,
> A lot of License-specific tests fail due to RoyaltyPolicyLAP, which requires allowance, payment, etc. related to commercial licenses. I thought these tests should only deal with licenses, so I used a mock RoyaltyPolicyLAP.
> 
> However, if we deem these tests more as integration tests, then I will use RoyaltyPolicyLAP. It will require dealing with LAP allowance and payments for minting licenses in the tests.